index.adoc

```diff
@@ -19,23 +19,10 @@ EARLY ACCESS PHASE
 
 [WARNING]
 ====
-This is preview software. We're gathering feedback to improve the service. It is not meant to be relied upon just yet.
-====
-
-= Roadmap
-
- - Web interface rework (hercules-ci.com)
- - Cachix integration
- - Impure tasks (CD)
- - Multi-agent support (cluster)
- - requiredFeatures support
- - Non-linux architecture support
- - Remove source code permission from the GitHub App
-
-= Known issues
+This is preview software. We're https://github.com/hercules-ci/support[gathering feedback] to improve the service. It is not meant to be relied upon just yet.
 
- - The identity token must be manually deleted when rekeying the agent token. https://github.com/hercules-ci/hercules-ci-agent/issues/8[issue #8]
-  - Agent can only build for a single github account (user / organization)
+Have a look at our https://github.com/hercules-ci/support/issues?q=is%3Aissue+is%3Aopen+label%3Aroadmap[roadmap] to know what (not) to expect.
```

Comment:
We could use `bug` label for known issues and `roadmap` for planned features, so that we can link those separately here.

Reply:
There's a fine line between bugs and features, if at all. Is this a blocker?